README.md

```diff
@@ -1,37 +1,148 @@
-# Ruby Class Variables And Class Methods
+# Class Variables and Class Methods
 
-## Outline
+## Objectives
 
-All objects are a combination of data and logic encapsulated by a singular object. We're used to seeing this on an instance level, on instances of classes.
+1. Define a class variable. 
+2. Define a class method. 
+3. Understand why and when to use class variables and methods. 
+4. Understand the scope of class variables and class methods. 
 
+## Introduction: Classes are Objects Too
+
+All objects are bundles of data and logic––or attributes and behavior. We understand this to be true of instances of a class. Each instances contains attributes or properties as well as methods that can enact behaviors. 
+
+For example, let's say we have a class, `Album`. Every individual album instance should have a property, `@release_date` and an instance method `#release_date` that exposes or reveals that property. 
+
+```ruby
+class Album
+
+  def release_date=(date)
+    @release_date = date
+  end
+  
+  def release_date
+    @release_date
+  end
 ```
-album.release_date = / @release_date
+
+Here we have an instance variable, `@release_date`, which can be set equal to a value using the `release_date=()` method, a setter method. Then, we have a getter method `release_date` that returns the value of `@release_date`. Now, we can execute the following:
+
+```ruby
+album = Album.new
+album.release_date = 1991
+album.release_date 
+  => 1991
+```
+
+What you might not know, however, is that the `Album` class itself is also an object. If our definition of an object is a bundle of code that contains attributes and behaviors, than the entire `Album` class itself absolutely qualifies. 
```

Comment:
I just generally like this introduction. Ground them to something they know, then push the boundry